QUEST_LV_0100.tsv

```diff
@@ -9510,48 +9510,48 @@ QUEST_LV_0100_20151001_009505	You are being lured to some mysterious power.
+QUEST_LV_0100_20151001_009535	You've obtained enough polluted red water from the old well. Pass the polluted red water to the villager.
 QUEST_LV_0100_20151001_009536	Obtain %s from the old well.
-QUEST_LV_0100_20151001_009537	The villager told you that you've obtaiined enough red water to pollute the food and he wants to tell you the location where Pharets hide the food. Talk to the villager.
+QUEST_LV_0100_20151001_009537	The villager told you that you've obtained enough red water to pollute the food and he wants to tell you the location where the Pharets hide the food. Talk to the villager.
 QUEST_LV_0100_20151001_009538	Pollute the fresh food with red water
-QUEST_LV_0100_20151001_009539	The villager told you that Pharets hide the stolen food among the fruit baskets. He told you that thare are some food that had been stolen long ago so only the fresh food should be polluted.
+QUEST_LV_0100_20151001_009539	The villager told you that Pharets hide the stolen food among the fruit baskets. He told you that there are some food that had been stolen long ago so only the fresh food should be polluted.
 QUEST_LV_0100_20151001_009540	Return to the villager
-QUEST_LV_0100_20151001_009541	You've found the fresh food from the suspicious baskets and polluted then with red water. Return to the villager.
+QUEST_LV_0100_20151001_009541	You've found the fresh food within the suspicious baskets and then proceeded to pollute it with red water. Return to the villager.
 QUEST_LV_0100_20151001_009542	The sculpture of peace
 QUEST_LV_0100_20151001_009543	Talk to the grandchild of the elder
-QUEST_LV_0100_20151001_009544	The grandchild of the elder seems to have some words to tell you after he went out to check whether Pharets have become gentle since the collapse of demonic totem. Talk to the grandchild of the elder.
+QUEST_LV_0100_20151001_009544	The grandchild of the elder seems to have some words to tell you after he went out to check whether the Pharets have become more gentle since the collapse of demonic totem. Talk to the grandchild of the elder.
 QUEST_LV_0100_20151001_009545	Talk to Divdirbys Withus
-QUEST_LV_0100_20151001_009546	The grandchild of the elder told you that although the demonic totem has been destroyed, the pharets were still very violent. Talk to Divdirbys, Withus if there's any way to make them calm.
-QUEST_LV_0100_20151001_009547	Divdirbys, Withus was actually enagaged in the research to create a sculpture which has opposite effects as compared to the demonic totem. Talk to Withus.
+QUEST_LV_0100_20151001_009546	The grandchild of the elder told you that although the demonic totem has been destroyed, the Pharets were still very violent. Talk to Divdirbys, Withus if there's any way to make them calm.
+QUEST_LV_0100_20151001_009547	Divdirbys, Withus was actually enagaged in research to create a sculpture which would have an opposite effect as compared to the demonic totem. Talk to Withus.
```

Comment:
Withus was actually engaged in research to create a sculpture which would oppose the effects of the demonic totem.